src/build/app/settings.rs

```diff
@@ -558,6 +563,19 @@ pub enum AppSettings {
     /// [`Arg::use_delimiter(false)`]: crate::Arg::use_delimiter()
     DontDelimitTrailingValues,
 
+    /// Disables colorized help messages.
+    ///
+    /// # Examples
+    ///
+    /// ```no_run
+    /// # use clap::{App, AppSettings};
+    /// App::new("myprog")
+    ///     .setting(AppSettings::DisableColoredHelp)
+    ///     .get_matches();
+    /// ```
+    #[cfg(feature = "color")]
+    DisableColoredHelp,
```

Comment:
This wasn't behind a feature flag before, should it be now?

Reply:
This wasn't a setting before. This is a new setting which is inverted.

Reply:
That is just semantics, driven by our policy that a flag has to always represent the "true" state.  Its us inverting an existing flag.  The inverted form of this flag was not conditional.

Reply:
Yeah, the previous one wasn't gated because the macro didn't support gates on the settings until recently when I added them.

Reply:
So long as the feature gate is used in an additive manner (which it is here) I don't see a conflict.